cli/src/main/java/org/jboss/pnc/bacon/cli/App.java

```diff
@@ -143,17 +143,16 @@ public int run(String[] args) {
                         .terminal(terminal)
                         .completer(systemRegistry.completer())
                         .parser(parser)
+                        .variable(LineReader.HISTORY_FILE, Constant.HISTORY)
+                        .variable(LineReader.HISTORY_FILE_SIZE, 50)
```

Comment:
is that last 50 entries in the history file? I'd like much more :)

Reply:
Er, sure, sorry just picked a random number :-)